src/modelbench/run.py

```diff
@@ -101,10 +103,13 @@ def benchmark(
     anonymize=None,
     parallel=False,
 ) -> None:
+    start_time = datetime.now(timezone.utc)
     suts = find_suts_for_sut_argument(sut)
-    benchmarks = [b() for b in BenchmarkDefinition.__subclasses__() if b.__name__ in benchmark]
-    benchmark_scores = score_benchmarks(benchmarks, suts, max_instances, debug, parallel)
+    benchmark = [b() for b in BenchmarkDefinition.__subclasses__() if b.__name__ == benchmark_name][0]
+    benchmark_scores = score_benchmarks([benchmark], suts, max_instances, debug, parallel)
```

Comment:
If we anticipate only one benchmark to be run at a time, maybe we can also simplify `score_benchmarks` and `score_a_sut` as well.

Reply:
I did look at that, but it involved changing a lot of code, so I thought I'd leave that for a future cleanup.

Reply:
Fair enough